gm4_cozy_campfires/pack.mcmeta

```diff
@@ -0,0 +1,20 @@
+{
+    "pack": {
+        "pack_format": 5,
```

Comment:
pack format will need to be changed to 6 for 1.16.2

Reply:
Fixed!